### DISCUSSION TITLE
Unable to run the interactive example of how distributions works

### DISCUSSION THREAD

**Discussion by RAppana** [gmdsi/GMDSI_notebooks#103] [Q&A] [answered]
I've posted the error I am getting when I try to import the bayes_helper function: 

![screenshot_bayes](https://github.com/gmdsi/GMDSI_notebooks/assets/33162513/7babc863-14c8-4a5a-b5af-a0da3db8a5fb)

**Comment by mnfienen** [comment]
hi @RAppana  - you need to be sure you are running in the environment you made for the class which has `pyemu` installed.

**Reply by RAppana** [reply]
That's right. I opened the jupyter notebook after activating the gmdsitut environment. Does this mean I need to install pyemu on my machine then?

**Reply by mnfienen** [reply]
hmmm....if you are opening the notebook in the gmdsitut environment and you were able to install that environment correctly then `pyemu` should import without an issue

**Reply by RAppana** [reply]
ALRIGHT, ALRIGHT, ALRIGHT. I moved the parent folder to a different directory. Maybe that is an issue?

**Reply by mnfienen** [reply]
the folder structure within the `GMDSI_notebooks` folder must be maintained for everything to work correctly. But if you move that top level folder around, there shouldn't be a problem. Have been able to get other notebooks to work correctly?

**Reply by RAppana** [reply]
I did not change the folder structure. I see that I have a problem loading pyemu and flopy. I'm not sure why but other tutorials with these dependencies aren't running either.

**Reply by mnfienen** [reply, marked as answer]
I think there are static paths to the pyemu and flopy packages, so you cannot move the folder once you create the environment. Can you put the folder back where you originally had it and try again? That should fix the problem

**Reply by RAppana** [reply]
Thank you so much, that worked.

**Reply by mnfienen** [reply]
awesome!
